pgvectorscale/src/access_method/guc.rs

```diff
@@ -37,4 +41,73 @@ pub fn init() {
         GucContext::Userset,
         GucFlags::default(),
     );
+
+    GucRegistry::define_float_guc(
+        unsafe { std::ffi::CStr::from_ptr("diskann.parallel_flush_rate".as_pg_cstr()) },
```

Comment:
I think this GUC is not a rate (higher = faster) but an interval (higher = slower), can you rename?